app_dart/test/service/scheduler_test.dart

```diff
@@ -154,6 +154,30 @@ void main() {
     );
   }
 
+  setUp(() {
+    logs = [];
+    log = Logger.detached('scheduler_test');
```

Comment:
Same comment; if we only use this to print when there's an error; why not just StringBuffer() and print later?